crates/cli/src/main.rs

```diff
@@ -275,6 +279,69 @@ mod linux {
     }
 }
 
+#[cfg(target_os = "linux")]
+mod flatpak {
+    use std::ffi::OsString;
+    use std::path::PathBuf;
+    use std::process::Command;
+    use std::{env, process};
+
+    /// Restarts outside of the sandbox if currently running within it
+    pub fn try_restart_to_host() {
+        if let Some(bin_dir) = get_flatpak_bin_dir() {
+            let mut args = vec!["/usr/bin/flatpak-spawn".into(), "--host".into()];
```

Comment:
Could the `/usr/bin/flatpak-spawn` be in a const for easier changes if needed ?

Reply:
It shouldn’t ever need to change, but I’ll do it anyways once I get the time.

Reply:
No need unless you're super keen to